Herbert/Framework/Enqueue.php

```diff
@@ -66,13 +66,15 @@ public function buildInclude($attrs, $footer)
 //            $attrs['src'] = ltrim($attrs['src'], '/');
 //        }
 
+        if (!isset($attrs['dep'])) $attrs['dep'] = [];
+
         if (pathinfo($attrs['src'], PATHINFO_EXTENSION) === 'css')
         {
             wp_enqueue_style($attrs['as'], $attrs['src']);
         }
         else
         {
-            wp_enqueue_script($attrs['as'], $attrs['src'], [], false, $footer);
+            wp_enqueue_script($attrs['as'], $attrs['src'], $attrs['dep'], false, $footer);
```

Comment:
I would use 'deps' as the key just like WP Codex uses. 

Since enqueue-ing styles has dependancies, I would set `$attrs['deps']` before css check. 
`$attrs['deps'] = (array) array_get($attrs, 'dep', []);`

Now all is needed is:
`wp_enqueue_style($attrs['as'], $attrs['src'], $attrs['deps']);`
and
'wp_enqueue_script($attrs['as'], $attrs['src'], $attrs['deps'], false, $footer);'